config/lobby/db/004_audit_bans

```diff
@@ -0,0 +1,23 @@
+start transaction;
+
+alter table banned_macs
+  add column mod_username varchar(40),
+  add column mod_ip varchar(45),
+  add column mod_mac char(28),
+  add constraint banned_macs_mod_mac_check check (char_length(mod_mac)=28);
+
+comment on column banned_macs.mod_username is 'The username of the moderator that executed the ban.';
+comment on column banned_macs.mod_ip is 'The IP address (v4 or v6) of the moderator that executed the ban.';
+comment on column banned_macs.mod_mac is 'The hashed MAC address of the moderator that executed the ban.';
+
+alter table banned_usernames
+  add column mod_username varchar(40),
+  add column mod_ip varchar(45),
+  add column mod_mac char(28),
+  add constraint banned_usernames_mod_mac_check check (char_length(mod_mac)=28);
+
+comment on column banned_usernames.mod_username is 'The username of the moderator that executed the ban.';
+comment on column banned_usernames.mod_ip is 'The IP address (v4 or v6) of the moderator that executed the ban.';
+comment on column banned_usernames.mod_mac is 'The hashed MAC address of the moderator that executed the ban.';
```

Comment:
Do we have mac + IP in case a moderator account is hacked?

Reply:
Not sure what you're asking here.  Could you elaborate a bit, please?

Reply:
We have private databases yes.  Not sure what is avail on server thus the above requests?

Reply:
Thanks for the nudge, @prastle.  Now I understand what @DanVanAtta was asking for.

Reply:
Asked another way, why record moderator IP address and MAC?  username uniquely identifies the moderator, why is ip or mac interesting to know?

Reply:
I believe the IP would make sense for the event of a hacked account, the hashed Mac wouldn't make a difference for this usecase IMO.

Reply:
I'm pretty sure I only added both IP and MAC because it was requested by the mod team in the feature request.  Reviewing the feature request again, I see that I misinterpreted one of @prastle's comments.  He was asking for the banee username to be recorded when they are banned by MAC so the mods can link username <-> MAC.  He was _not_ asking for the moderator MAC to be recorded.  So I'll remove the `mod_mac` column.

I think what @prastle was originally requesting has value, but it's distinct enough that I think it would be better to do it in a separate PR immediately following this one.  If we're going to record the username when banned by MAC, we might as well record the MAC when banned by username.  In that case, I wonder if there's a way to combine the `banned_macs` and `banned_usernames` tables?

Reply:
Prob should include both @ssoloff unless you think the mods will take up to much space.  But yes we need the banned booted info of the player.  Mac/IP/Name and why and what mod banned.  Identical to what would appear if we copy paste his info in lobby.

Reply:
Mutes i would ignore. unless over 20 min mute

Reply:
@prastle Understood; I won't remove `mod_mac`.  28 characters per record shouldn't exhaust our disk space. :smile:

Reply:
I'll add the audit information for mutes in a follow-up PR.  It will just be easier to do it for all mutes regardless of duration.  Again, we're not talking about a lot of additional disk usage here.

Reply:
Oki doki

Reply:
Also @DanVanAtta  the reason for mod mac etc inclusion is that we have quite a few afk mods.  This makes us all accountable.